pkg/mcp/details.go

```diff
@@ -55,14 +68,16 @@ func (sm *SessionManager) GetServerDetails(ctx context.Context, serverConfig Ser
 	}
 
 	var mcpEvents []types.MCPServerEvent
-	for _, event := range events.Items {
+	for _, event := range append(deploymentEvents.Items, podEvents...) {
```

Comment:
Should we sort these based on creation time?

Reply:
Yes, good call, I will add that

Reply:
Fixed